tool/lint.sh

```diff
@@ -3,9 +3,15 @@
 set -e
 
 bad=$(egrep -r ',\s*@Cached' src || exit 0)
-
 if [ -n "$bad" ]; then
 	echo "@Cached arguments should be on their own line"
 	echo "$bad"
 	exit 1
 fi
+
+bad=$(egrep -r 'computeIfAbsent' src || exit 0)
+if [ -n "$bad" ]; then
+	echo "Use get() + putIfAbsent() instead of not scaling computeIfAbsent()"
```

Comment:
For a giving key, all accesses are serialized, because ConcurrentHashMap.computeIfAbsent() always takes the lock of the bucket if the key is present. This for the Charset case actually prevents scaling on a benchmark rather quickly as the number of threads increases.